src/DotNetty.Transport/Channels/IChannelInboundHandler.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Transport.Channels
+{
+    /// <summary>
+    /// <see cref="IChannelHandler"/> which adds callbacks for state changes. This allows the user
+    /// to hook in to state changes easily.
+    /// </summary>
+    public interface IChannelInboundHandler : IChannelHandler
```

Comment:
remove.
1. It goes against the model we've inherited from netty 5.0
2. If we were to do it, we would bring the rest of the related artifacts and usages from netty 4.1.